src/workers/VolumeLoaderContext.ts

```diff
@@ -173,9 +174,25 @@ class VolumeLoaderContext {
   /** Close this context, its worker, and any active loaders. */
   close(): void {
     this.workerHandle.close();
-    this.activeLoader?.close();
   }
 
+  private handleEvent(e: ChannelLoadEvent | MetadataUpdateEvent): void {
+    const loader = this.loaders.get(e.loaderId);
+    if (loader) {
+      if (e.eventType === WorkerEventType.CHANNEL_LOAD) {
+        if (this.throttleChannelData) {
+          throttle(() => loader.onChannelData(e));
+        } else {
+          loader.onChannelData(e);
+        }
+      } else if (e.eventType === WorkerEventType.METADATA_UPDATE) {
+        loader.onUpdateMetadata(e);
+      }
+    }
+  }
+
+  private;
```

Comment:
What does this standalone `private;` do?

Reply:
What indeed?

(Nothing. It's gone.)